internal/auth/authz/k8s.go

```diff
@@ -0,0 +1,66 @@
+package authz
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"io"
+	"net/http"
+
+	k8sAuthorizationV1 "k8s.io/api/authorization/v1"
+)
+
+type K8sAuthZ struct {
+	ApiUrl string
+}
+
+func createSSAR(resource string, verb string) ([]byte, error) {
+	ssar := k8sAuthorizationV1.SelfSubjectAccessReview{
+		Spec: k8sAuthorizationV1.SelfSubjectAccessReviewSpec{
+			ResourceAttributes: &k8sAuthorizationV1.ResourceAttributes{
+				Verb:     verb,
+				Group:    "flightctl.io",
```

Comment:
Can we make this as a configuration? "flightctl.io" is finalized?

Reply:
Just my wild guess.

It seems weird to me to have the group/kind configurable.
@fzdarsky  WDYT ?

Reply:
The domain isn't reserved / finalized, no. I'd suggest going with it for now, though.